index.js

```diff
@@ -4,7 +4,7 @@ const LambdaEnvVars = require('lambda-env-vars');
 const HoldRequestConsumerModel = require('./src/models/HoldRequestConsumerModel');
 const HoldRequestConsumerError = require('./src/models/HoldRequestConsumerError');
 const ApiServiceHelper = require('./src/helpers/ApiServiceHelper');
-const SCSBApiHelper = require('./src/helpers/SCSBApiHelper');
+const HoldingLocationHelper = require('./src/helpers/HoldingLocationHelper');
```

Comment:
Minor thing, but this helper feels more like a router/dispatcher that happens to leverage holding location to determine where to route/dispatch the hold-request. Have you considered a name like HoldRequestDispatcher?

Reply:
I like that idea. Should it be in a separate subdirectory, i.e. `src/dispatchers/HoldRequestDispatcher`?

Reply:
The way I was thinking about it, there's only one dispatcher, so I wouldn't add a directory. One might argue that the components that you dispatch *to* (i.e. SCSBApiHelper, OnSiteHoldRequestHelper) could live in their own subdirectory, but that starts to feel like a bigger architectural change to the app, and I don't think we want to go there..